src/openfermion/hamiltonians/_jellium.py

```diff
@@ -89,13 +89,16 @@ def plane_wave_kinetic(grid, spinless=False, e_cutoff=None):
     return operator
 
 
-def plane_wave_potential(grid, spinless=False, e_cutoff=None):
+def plane_wave_potential(grid, spinless=False, e_cutoff=None,
+                         non_periodic=False, period_cutoff=None):
     """Return the e-e potential operator in the plane wave basis.
 
     Args:
         grid (Grid): The discretization to use.
         spinless (bool): Whether to use the spinless model or not.
         e_cutoff (float): Energy cutoff.
+        non_periodic (bool): If the system is non-periodic.
+        period_cutoff (float): Period cutoff.
```

Comment:
Perhaps state what the default is here for people that want to specify non_periodic but not the cutoff

Reply:
Done - also other instances.